.gitignore

```diff
@@ -0,0 +1,4 @@
+vendor
+
+PixelUnion/Sniffs
+PixelUnion/Tests/Methods
```

Comment:
☝️These are temporary to keep the PR clean (I could have just not committed the files buuut...)